### ISSUE TITLE
Firmware flashing error

### ISSUE THREAD

**Issue by urherenow** [AcK77/AmiiBomb-uino#10]
First off, I have no clue about the specifics of my arduino. It's obviously a clone and goes by the name EDarduino. The arduino suite did not even have drivers that would install for my unit. I had to download drivers for CH340, which properly made it a com (4) port.

The internal flasher errors out no matter which model I choose to flash. Any suggestions or does this model need it's own firmware? The other flashing method didn't spit out any errors, but I'm not sure if it's working at all. I haven't recieved my NTAG stickers yet and there doesn't seem to be anything at all I can do with the included key fob or card, to include telling me what kind of NTAG it is. I thought at least it would detect an NTAG and give me an error that it's not the correct kind. But no indication at all...

I am using the v0.3

**Comment by cspencer49519** [comment]
There are some example sketches in the MFRC522 library that will allow you to test communications between your RF module/NTAG and your arduino. Have you tried to test these example sketches to verify that everything is connected properly? Also, did you flash the firmware as someone suggested on GBA?

**Comment by peacepenguin** [comment]
Check out this readme and associated fork that I've created from this repo. The readme describes my process for manually installing the Arduino IDE and making the arduino ide work with my "clone", esp32 board. 

Most important was to change the pins used by amiibombuino firmware, the change for my esp32 board is also available in my forked repository.

Instructions for esp32:
https://github.com/peacepenguin/AmiiBomb-uino/blob/master/README-esp32.md

Modified arduino sketch that allows the esp32 to work with amiibomb windows app:
https://github.com/peacepenguin/AmiiBomb-uino/blob/master/AmiiBombuino/AmiiBombuino-esp32.ino

I want to make some more commits to getting the esp32 in the dropdown and flashable from within amiibomb, but for now you should be able to take my work in progress manual steps to get your clone duino working.

Biggest challenge will be:
1. Mapping the ports correctly for interfacing to the mfrc522 board
2. Getting the cloneduino working in the arduino IDE (this should be easy if it is a true "clone" duino, esp32 requires manuall steps, so knowing that i had success with esp32 should be encouragement enough that yiu can get a clone working!)

**Comment by AcK77** [comment]
Hi

@urherenow if you want to know if AmiiBombuino was flashed good inside your Arduino clone, just try to dump an Amiibo and you have a status at the bottom of the windows (a Red or Green dot). 
I can't support all of Arduino Clone, it's too much works to do that... You can simply use XLoader if the internal flasher don't works or use the Arduino IDE and compile the AmiiBombuino source code to your clone. 

@peacepenguin 
Great job done!! We can include the ESP32 to AmiiBomb together if you want ;)! It's better to have an all in one app instead of multiple fork :)! Let me know if you are interested to add support of the ESP32 inside AmiiBomb. It's certainly possible to add the ability of use the ESP32 without the Serial, send and receive informations through TCP connection by example. With that, we can get a wireless AmiiBombuino :)!

**Comment by peacepenguin** [comment]
Yes I would love to get the esp32 integrated! I just forked for now to have an area to publicly experiment with your code until I got something actually working to offer back to the main repo!

Wireless amiibomb sounds awesome, great project to push these esp32 boards to their full potential!

I finally got VS installed to actually attempt compiling from source to really start experimenting now that I know the esp32 board can work when manually flashing via the arduino ide!
